### ISSUE TITLE
training stage

### ISSUE THREAD

**Issue by OrangeZz0331** [puckikk1202/difffacesketch2023#3]
I want to train my pre trained model, but I seem to be encountering some issues.After setting your origin dataset path and output path respectively,start training,butBut it seems to be stuck and hasn't generated a model yet,Can you tell me in detail how to train it？

And this is my project directory.Store facial images in /DiffFaceSketch/datasets/celeba_img_nobg，sketch iamges in /DiffFaceSketch/datasets/celeba_sketch_nobg.The pre trained model to be generated is stored in /DiffFaceSketch/zkc_premodels.

Thank you very much!!!
![project directory](https://github.com/puckikk1202/difffacesketch2023/assets/148956633/9fde7b48-1a7a-4e33-8f6d-99ae5798d02e)
![stuck](https://github.com/puckikk1202/difffacesketch2023/assets/148956633/7bf7fdb9-c503-4da9-aaf6-6dd7e5edbac1)

**Comment by puckikk1202** [comment]
From your log, it seems that the training has already been initialized. Can you print more information in the training loop to see where it gets stuck?

**Comment by OrangeZz0331** [comment]
It got stuck on this interface, no new things were generated, and the backend GPU was running continuously.
![stuck](https://github.com/puckikk1202/difffacesketch2023/assets/148956633/abceb1bf-a911-4332-8ae0-daf17b326070)

**Comment by OrangeZz0331** [comment]
Can you share your sketch database for us to train models？I want to try it.Thank you very much!&nbsp;


橙橙子
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月30日(星期四) 下午5:20
收件人: "puckikk1202/difffacesketch2023";
抄送: "Author";
主题: Re: [puckikk1202/difffacesketch2023] training stage (Issue #3)





 
Yes, the log shows the model is initialized and started to train. I think you may try to print more log information in the training loop to check whether your data was loaded correctly, or the loss was updated correctly, etc.
 
&mdash;

**Comment by OrangeZz0331** [comment]
I'm sorry to bother you, but I really need your help！
I want to know if I need to train “train_ae.py” before training “train_sketchdiff.py”？
What should I do when I encounter the following error while running “train_ae.py” ？
Looking forward to your reply~
